combine m12 and m13 json config files

dawsonhemphill opened the pull request.

dawsonhemphill committed:
combine m12 and m13 json config files

wmmc88 commented on odrive/torque_test/m12_13.json:
```diff
@@ -0,0 +1,499 @@
+            "mode": 0,
+            "phase_offset_float": 0.5678834915161133,
+            "phase_offset": 2019,
+            "pre_calibrated": false,
```
true on other axis

matthew-reynolds replied:
not resolved in 35299e41b48b630da2b45f66460a1852e762b9fc. dunno if we do or dont want enc precalibrated, but it should at least be consistent

@dawsonhemphill @wraftus

wraftus replied:
Should leave it as false I think, m12 hasn't actually been calibrated since it got reassembled iirc, so the encoder calib is likely wrong

matthew-reynolds replied:
#11

wmmc88 commented on odrive/torque_test/m12_13.json:
```diff
@@ -0,0 +1,499 @@
+            "inertia": 0.0,
+            "input_filter_bandwidth": 2.0,
+            "input_mode": 1,
+            "load_encoder_axis": 1,
```
```suggestion
            "load_encoder_axis": 0,
```

matthew-reynolds replied:
resolved in 35299e41b48b630da2b45f66460a1852e762b9fc

wmmc88 reviewed (changes requested):
Found some differences by diffing. Also the formatting is different than the other jsons. could u go in vscode and click format?

wraftus committed:
Retune m13's controller and fix copying issues \
o Bumped vel_limit to 7 from 4.5 for M0
o Bumped current_control_bandwidth to 150 from 100 for M0
o Retuned controller for M0

wraftus commented:
Retuned M13's controller and played with the `vel_limit` and `current_control_bandwidth`, notes in commit message.

I also renamed the combined file `m12_13` to `m13_m12` to reflect that m13 goes in M0, this naming convention should make things a little easier on our brains
